apps/judicial-system/web/src/routes/Prosecutor/Overview/Overview.tsx

```diff
@@ -343,17 +343,20 @@ export const Overview: React.FC = () => {
                     </Box>
                   </AccordionItem>
                 )}
-                <AccordionItem
-                  id="id_6"
-                  label="Athugasemdir vegna málsmeðferðar"
-                  labelVariant="h3"
-                >
-                  <Text>
-                    <span className={styles.breakSpaces}>
-                      {workingCase.comments}
-                    </span>
-                  </Text>
-                </AccordionItem>
+                {features.includes(Feature.CASE_FILES) &&
+                  !!workingCase.caseFilesComments && (
+                    <AccordionItem
```

Comment:
Shouldn't the old workingCase.comments remain as they were and the new workingCase.caseFilesComments simply be added?